src/Mapper.cpp

```diff
@@ -1346,3 +1346,223 @@ map<int, CGRANode*>* Mapper::getReorderPath(map<CGRANode*, int>* t_path) {
   return reorderPath;
 }
 
+
+// save the mapping results to json file for subsequent incremental mapping
```

Comment:
`// save the mapping results to json file for subsequent incremental mapping` -> `// Saves the mapping results to json file for subsequent incremental mapping.`

Same to the other comments, thanks!

Reply:
Has been corrected